src/ci/ci-status.js

```diff
@@ -39,35 +39,37 @@ const statusLine = (_status, name) => {
   return `⦿ ${name} - ${status} - ${url}`;
 };
 
+const parseRef = ref => {
+  const pullPattern = /^#?\d{1,10}$/;
```

Comment:
It's theoretically possible for a commit sha to consist of only numbers too.  This might get weird

Reply:
pushed it